libs/elements/layout/buttons/src/lib/NormalButton/normal-button.component.html

```diff
@@ -0,0 +1 @@
+<p>normal-button works!</p>
```

Comment:
Only one reusable normal button component is needed. So we can delete this one (component)

Reply:
Done,